index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Document</title>
+ <link rel="stylesheet" href="./index.css">
+ <link rel="stylesheet" href="./navbar.css">
+</head>
+<body>
+<div id="navbar" >
+  <div id="menu">
+   <a href="#projects" id="profile-link"><p>Projects</p></a>
+  </div>
+  <div id="github">
+   <a href="https://github.com/IsmaelMastronardi" target="_blank"id="profile-link"><p>Github</p></a>
+  </div>
+</div>
+<div class="resto">
+  <div id="welcome-section">
+    <h1 id="textwel">Welcome to my personal portfolio</h1>
+  </div>
+   <div id="projects">
+    <a href="https://www.freecodecamp.com"> <img class="project-tile" src="./images/pr2mod.png" alt=""></a>
+    <a href="https://www.freecodecamp.com"> <img class="project-tile" src="./images/beli (1).jpg" alt=""></a>
+    <a href="https://www.freecodecamp.com"> <img class="project-tile" src="./images/hrefe mod(1).jpg" alt=""></a>
+    <a href="https://www.freecodecamp.com"> <img class="project-tile" src="./images/pr4sized.jpg" alt=""></a>
+   
+    
```

Comment:
- [ ] kindly consider phasing your comments correctly as suggestions instead of orders to make your suggestions friendly
eg :
```
* This is a polite suggestion: *"I think it would be better to use semantic tags"*
* This reads as an order: *"Use semantic tags"*
```